src/Ledger/Foreign/HSLedger.agda

```diff
@@ -166,20 +173,20 @@ instance
 
       from' : F.TxBody → TxBody
       from' txb = let open F.TxBody txb in record
-        { txins      = from ⦃ Convertible-FinSet ⦃ Coercible⇒Convertible ⦄ ⦄ txins
-        ; txouts     = from txouts
-        ; txcerts    = []
-        ; mint       = ε -- since simpleTokenAlgebra only contains ada mint will always be empty
-        ; txfee      = txfee
-        ; txvldt     = coerce txvldt
-        ; txwdrls    = ∅ᵐ
-        ; txup       = nothing
-        ; txADhash   = nothing
-        ; netwrk     = nothing
-        ; txsize     = txsize
-        ; txid       = txid
-        ; txvote     = []
-        ; txprop     = []
+        { txins    = from ⦃ Convertible-FinSet ⦃ Coercible⇒Convertible ⦄ ⦄ txins
```

Comment:
This change was because this is a `.agda` file, not a `.lagda` file and I think @WhatisRT prefers to not have extra indents in `.agda` files (although think it would be simpler, and would prefer, if we could use one indentation style throughout---namely, a style that looks good in both emacs and the generated pdf).

Reply:
This is an instance where cleanup was undone: note that `txdonation` at the bottom is two characters longer so now the equality signs aren't aligned anymore.